Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -312,6 +316,9 @@ WI.FontStyles = class FontStyles
             }
         }
 
+        // returns true if the CSS property was authored with `!important` in its value (which exists on WI.CSSProperty._rawValue)
+        resultProperties.isImportant = style.domNodeStyle.effectivePropertyForName(name)?.important;
```

Comment:
Please rename `resultProperties` to `resultProperty` inside this method. The plural form is a typo from the original patch. This method deals with a single property.